sample-app/screens/HomeScreen.js

```diff
@@ -227,12 +344,30 @@ export default class HomeScreen extends React.Component<Props, State> {
               onPress={() => this.makeGooglePayPayment(false)}
             />
           )}
+          <View style={styles.spacing} />
+          {isIos ? (
+            <ApplePayButton
+              style={styles.payButtonStyle}
+              setThemeStyle={ButtonTheme.Dark}
+              onPayPress={() => this.makeAppleNativePayment() }
+            />
+          ) : (
+            <GooglePayButton
+              style={styles.payButtonStyle}
+              onPayPress={() => this.makeGoogleNativePayment() }
+            />
+          )}
         </View>
       </View>
     );
   }
 }
 
+const ButtonTheme = {
```

Comment:
This should be in the SDK, not in the sample

Reply:
👍